tests/lib/rules/require-default-props.js

```diff
@@ -1306,7 +1306,26 @@ ruleTester.run('require-default-props', rule, {
         column: 3
       }]
     },
-
+    //
+    // component with no pros followed by a failing component
```

Comment:
I was just following the pattern in the rest of the file, I have deleted it now though, since it does seem unnecessary 👍